client/i18n/locales/english/translations.json

```
@@ -95,7 +95,8 @@
     "update-card": "Update your card",
     "donate-now": "Donate Now",
     "confirm-amount": "Confirm amount",
-    "skip-advertisement": "Skip Advertisement"
+    "skip-advertisement": "Skip Advertisement",
+    "back": "Back"
```

```suggestion
    "back": "Back",
    "close-dialog": "Close dialog"
```

You might want to add this right after "close" on line 7? I just added it here since that's the only place github would let me add this suggestion in "buttons".